HomeTask1/Task1_1.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func median(i []int) float64 {
+	sort.Ints(i)
+	Num := len(i) / 2
+	if Odd(len(i)) {
```

Comment:
There's no need for this function: we can check the condition just in place. Please address.